src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,13 +1,23 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validation.ValidPassword;
+
+@ValidPassword
 public class UserRequestDto {
     private String email;
     private String password;
+    @NotNull
+    private String repeatPassword;
```

Comment:
```suggestion
    private String repeatPassword;
```
![image](https://user-images.githubusercontent.com/20326858/234533297-fe8b416d-dc72-4dc5-acbd-0957a089c119.png)